README.md

```diff
@@ -20,143 +20,20 @@ _Thanks to the [Better Jinja](https://github.com/samuelcolvin/jinjahtml-vscode)
-```
-{{ content.post_body }}
-{{ content.blog_post_author }}
-{{ group }}
-{{ next_page_num }}
-etc.
-```
-_NOTE_: Some of these variables are nested
-![Nested variables](https://cdn2.hubspot.net/hubfs/2359872/IMPORTANT/DONOTDELETE/hubl-language-extension/content..gif)
-
-`hubldoc`
-```
-<!doctype html>
-<!--[if lt IE 7]> <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="{{ html_lang }}" {{ html_lang_dir }}> <![endif]-->
-<!--[if IE 7]>    <html class="no-js lt-ie9 lt-ie8" lang="{{ html_lang }}" {{ html_lang_dir }}>        <![endif]-->
-<!--[if IE 8]>    <html class="no-js lt-ie9" lang="{{ html_lang }}" {{ html_lang_dir }}>               <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js" lang="{{ html_lang }}" {{ html_lang_dir }}> <!--<![endif]-->
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-        <meta name="author" content="{{ meta_author }}">
-        <meta name="description" content="{{ page_meta.meta_description }}">
-        <title>{{ page_meta.html_title }}</title>
-        {% if site_settings.favicon_src %}<link rel="shortcut icon" href="{{ site_settings.favicon_src }}" />{% endif %}
-        {{ standard_header_includes }}
-    </head>
-    <body>
-        ${stuff}
-        {{ standard_footer_includes }}
-    </body>
-</html>
-```
-`hublblog`
-```
-{% if is_listing_view %}
-    <!-- Markup for blog listing template -->
-    {% for content in contents %}
-        <h2><a href="{{content.absolute_url}}">{{ content.name }}</a></h2>
-        {% if content.post_list_summary_featured_image %}
-            <a href="{{content.absolute_url}}">
-                <img src="{{ content.post_list_summary_featured_image }}" alt="{{ content.featured_image_alt_text }}">
-            </a>
-        {% endif %}
-        {{ content.post_list_content|safe }}
-    {% endfor %}
-{% else %}
-    <!-- Markup for blog post template -->
-    <h1>{{ content.name }}</h1>
-    <a href="{{ group.absolute_url }}/author/{{ content.blog_post_author.slug }}">{{ content.blog_post_author.display_name }}</a>
-    {{ content.publish_date_localized }}
-    {{ content.post_body }}
-    {% for topic in content.topic_list %}
-        <a href="{{ blog_tag_url(group.id, topic.slug) }}">{{ topic.name }}</a>{% if not loop.last %},{% endif %}
-    {% endfor %}
-    {% blog_comments "blog_comments" overrideable=False, label='Blog Comments' %}
-{% endif %}
-```
-[`request.<variables>`](https://designers.hubspot.com/docs/hubl/hubl-supported-variables#http-request-variables)
-```
-{{ request.cookies }}
-{{ request.domain }}
-{{ request.full_url }}
-etc.
-```
-[`Email Required Template Variables`](https://designers.hubspot.com/docs/hubl/hubl-supported-variables#required-email-template-variables)
-```
-{{ site_settings.company_city }}
-{{ site_settings.company_name }}
-{{ unsubscribe_link }}
-{{ unsubscribe_link_all }}
-etc.
-```
-
-__Custom Module Fields__
```

Comment:
I think adding custom module fields back is to the readme is kind of a must, otherwise it's not mentioned at all.